src/test/java/eu/dissco/core/translator/terms/specimen/identification/taxonomy/SpecimenNameTest.java

```diff
@@ -29,54 +30,24 @@ void testRetrieveFromDWCA() {
   @Test
   void testRetrieveFromDWCAExtension() {
     // Given
-    var unit = MAPPER.createObjectNode();
-    var extension = MAPPER.createObjectNode();
-    var identificationUnverified = MAPPER.createObjectNode();
-    identificationUnverified.put("dwc:scientificName", "SpecimenNameUnVerified");
-    identificationUnverified.put("dwc:identificationVerificationStatus", "0");
     var identificationVerified = MAPPER.createObjectNode();
     identificationVerified.put("dwc:scientificName", "SpecimenNameVerified");
     identificationVerified.put("dwc:identificationVerificationStatus", "1");
-    var identifications = MAPPER.createArrayNode();
-    identifications.add(identificationVerified);
-    identifications.add(identificationUnverified);
-    extension.set("dwc:Identification", identifications);
-    unit.set("extensions", extension);
 
     // When
-    var result = specimenName.retrieveFromDWCA(unit);
+    var result = specimenName.retrieveFromDWCA(identificationVerified);
 
     // Then
     assertThat(result).isEqualTo("SpecimenNameVerified");
   }
 
-  @Test
-  void testRetrieveFromDWCAExtensionNoVerified() {
-    // Given
-    var unit = MAPPER.createObjectNode();
-    var extension = MAPPER.createObjectNode();
-    var identificationUnverified = MAPPER.createObjectNode();
-    identificationUnverified.put("dwc:scientificName", "SpecimenNameUnVerified");
-    identificationUnverified.put("dwc:identificationVerificationStatus", "0");
-    var identifications = MAPPER.createArrayNode();
-    identifications.add(identificationUnverified);
-    extension.set("dwc:Identification", identifications);
-    unit.set("extensions", extension);
-
-    // When
-    var result = specimenName.retrieveFromDWCA(unit);
-
-    // Then
-    assertThat(result).isNull();
-  }
-
   @Test
   void testRetrieveFromABCDEFG() {
     // Given
     var specimenNameString = "SpecimenName";
     var unit = MAPPER.createObjectNode();
     unit.put(
-        "abcd-efg:identifications/identification/0/result/mineralRockIdentified/classifiedName/fullScientificNameString",
+        "result/mineralRockIdentified/classifiedName/fullScientificNameString",
```

Comment:
I hate to suggest more tests, but it might be good to test more for the correct term being returned when you're searching multiple attributes. 
e.g. for abcd, the program first checks "taxonIdentified/scientificName/fullScientificNameString", then "mineralRockIdentified/classifiedName/fullScientificNameString". Might be good to test that the expected term is being returned if both fields are populated.

(this might not be the best example to bring up this point, since you SHOULDN'T have a taxon identification and a mineral identification in the same archive, but I wouldn't put it past some data sources...)

Reply:
Added